syft/messaging/message.py

```diff
@@ -149,9 +164,9 @@ def contents(self):
         self.message and self.return_ids, which allows for more efficient simplification (we don't have to
         simplify return_ids because they are always a list of integers, meaning they're already simplified)."""
 
-        message = (self.cmd_name, self.cmd_owner, self.cmd_args, self.cmd_kwargs)
+        message = (self.action.name, self.action.operand, self.action.args, self.action.kwargs)
 
-        return (message, self.return_ids)
+        return (message, self.action.return_ids)
 
     @staticmethod
     def simplify(worker: AbstractWorker, ptr: "OperationMessage") -> tuple:
```

Comment:
Are we planning to keep "simplify" and "detail' around long term given Protobuf?

Reply:
No, but they still need to work for the time being since Protobuf serialization isn’t complete yet.